secboot/secboot_sb_test.go

```diff
@@ -967,6 +967,15 @@ func (s *secbootSuite) TestResealKey(c *C) {
 			TPMPolicyAuthKeyFile: mockTPMPolicyAuthKeyFile,
 		}
 
+		numMockSealedKeyObjects := len(myParams.KeyFiles)
+		mockSealedKeyObjects := make([]*sb_tpm2.SealedKeyObject, 0, numMockSealedKeyObjects)
+		for range myParams.KeyFiles {
+			mockSealedKeyFile := filepath.Join("test-data", "keyfile")
```

Comment:
How is this keyfile generatd?

Reply:
It is a copy of https://github.com/snapcore/secboot/blob/master/internal/compattest/testdata/v1/key (i.e. any valid looking key that can be read into the sealed key object struct)

Reply:
@mvo5 ping